addons/payment_adyen/controllers/main.py

```diff
@@ -29,7 +29,7 @@ def adyen_return(self, **post):
 
     @http.route([
         '/payment/adyen/notification',
-    ], type='http', auth='none', methods=['POST'])
+    ], type='http', auth='none', methods=['POST'], csrf=False)
```

Comment:
looking at https://github.com/odoo/odoo/commit/3dafb19c96 and https://docs.adyen.com/display/TD/Set+up+notifications it has to be set manually on adyen